util/dvsim/dvsim.py

```diff
@@ -310,6 +310,12 @@ def parse_args():
                               'of categories from: {}.'.format(
                                   ', '.join(_LIST_CATEGORIES))))
 
+    parser.add_argument("--interactive",
+                        action='store_true',
+                        help=('If true (specified), dvsim launches task without'
+                              ' the ongoing status screen and allows user '
+                              'inputs.'))
```

Comment:
How about:

```suggestion
                        help=('Launch the job in non-GUI interactive mode'
                              ' accepting manual user inputs.'))
```

We need to clarify that this is not the same as GUI mode.